Source/Dafny/DafnyPipeline.csproj

```diff
@@ -1,23 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Target Name="RunCoco" BeforeTargets="PreBuildEvent" >
+  <Target Name="RunCoco" BeforeTargets="PreBuildEvent">
     <Exec Command="dotnet tool restore" />
+    <Exec Command="dotnet --info" />
```

Comment:
Is this --info line needed long-term?

Reply:
No, but I thought it could be useful in future debugging, so I left it in.